components/layout/blog-post/blog-post.hbs

```diff
@@ -1,26 +1,25 @@
 <div class="section--bg-prim-blue">
 
-    <div class="blog-post-hero section__inner">
-        <h1 class="blog-post-hero__title">If your team isn’t learning,
+    <div class="post-hero section__inner">
+        <h1 class="post-hero__title">If your team isn’t learning,
             they’re falling behind. Fix it.</h1>
-        <div class="blog-post-hero__blog-author">
-            <div class="blog-post-hero__blog-author-img-container">
-                <img class="blog-post-hero__blog-author-img-container__author-image"
+        <div class="post-hero__author">
+            <div class="post-hero__author-img-container">
+                <img class="post-hero__author-img-container__author-image"
```

Comment:
Something else I just saw here, you've chained two elements together with __. This probably just needs to be `post-hero__author-image`